src/metatrain/experimental/nanopet/tests/test_regression.py

```diff
@@ -44,11 +44,11 @@ def test_regression_init():
 
     expected_output = torch.tensor(
         [
-            [-0.031101370230],
-            [0.057578764856],
-            [0.023473259062],
-            [-0.040009934455],
-            [0.019827004522],
+            [-0.371612131596],
+            [-0.114121928811],
+            [0.010294607840],
+            [-0.210758388042],
+            [-0.044689975679],
```

Comment:
The reference value changed because you removed the bias, right?

Reply:
Yes, and also because adding one more test that is executed before this one (in this case the one-atom-structure test) changes the random initialization of the parameters, given that one more random model is initialized before the one in this test